### ISSUE TITLE
how to filter inside parent's names?

### ISSUE THREAD

**Issue by leocongote** [angular-gantt/angular-gantt#407]
When i use filter row it filters the rows(children) and it brings the parents too, but if i try to filter the specific name of one Parent it doesn't work because i guess it is only filtering the rows-children.

There is a way to filter inside parent's names?

Thanks, 

Great Job!!!

**Comment by Kize** [comment]
Hi, 

I agree with @leocongote, the automatic filter-row function searches only in children rows, not parents.

 If we take the https://www.angular-gantt.com/demo/ as example, we can't filter the rows  "Development", "Hosting" or "Server".

So I'm also looking for a way to filter rows by parents too.

Thanks

**Comment by Kize** [comment]
Hello,

I solved this issue but with a little trick and a `filter-row-comparator`.

The idea is that every child on the lower level needs to know the `name` property of every parent `row`. 
We need to filter only rows on the lower level, because the attribute `keep-ancestor-on-filter-row` of gantt-tree will dislay parents for us.

So, add a new property to each row without child. 

This property (I called it `tags`) contains a string with its own name, and the name of each parent.


Then i use this filter-row-comparator : 
```javascript
filterRowComparator: function (actual, expected) {
            if (expected.name == "")
              return actual;
            if (actual.tags) {
              var tagsUpped = actual.tags.toUpperCase(),
                  searchUpped = expected.name.toUpperCase();
              if (tagsUpped.includes(searchUpped))
                return actual
            }
          }
```
 
It works find, but note that `#includes()` works only on recent browsers, and the comparaison excludes accents. `"é" != "e"`
